### DISCUSSION TITLE
[Feature] Request:  Add a search field in "search" -> "people" -> "view all" tab so you can search by name.

### DISCUSSION THREAD

**Discussion by WoutAndAbout** [immich-app/immich#11102] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add a search field in "search" -> "people" -> "view all" tab so you can search by name.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5206

As requested by the template, please search for duplicates before opening a request.

**Comment by WoutAndAbout** [comment]
This is not a duplicate since this is for mobile and not for web.
